src/test/java/com/google/sps/tests/NewsTest.java

```diff
@@ -0,0 +1,152 @@
+package com.google.sps.servlets;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+import java.util.List;
+import java.util.ArrayList;
+import com.google.sps.data.Article;
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.FileReader;
+import java.io.FileInputStream;
+import java.net.HttpURLConnection;
+import java.util.Map;
+import java.util.HashMap;
+
+
+@RunWith(JUnit4.class)
+public final class NewsTest {
+
+  public final static String FULL_HTML_PATH = "src/test/java/com/google/sps/data/fullNewsHTML.txt";
```

Comment:
[Nit] Since you use the string 'src/test/java/com/google/sps/data' in all of the constants here I would just create a constant for that.

`public final static String HTML_PATH_PREFIX = "src/test/java/com/google/sps/data"`

Then you can construct the other constants by just concatenating onto the end of that prefix.